core/templates/pages/collection-editor-page/editor-tab/collection-node-creator.directive.ts

```diff
@@ -27,6 +27,8 @@ require(
 require('services/alerts.service.ts');
 require('services/site-analytics.service.ts');
 require('services/validators.service.ts');
+// eslint-disable-next-line max-len
+require('pages/collection-editor-page/services/collection-node-creator-backend-api.service.ts');
```

Comment:
```suggestion
require(
  'pages/collection-editor-page/services/' +
  'collection-node-creator-backend-api.service.ts');
```

Avoid using pragama!